### PR TITLE
LPD-7012 - Add functional tests

### PR DESCRIPTION
**Pull request by ilzamcmed**

### CODE REVIEW

**Comment by ilzamcmed** [comment]
ci:test:playwright

File: modules/test/playwright/utils/portletUrls.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 export const PORTLET_URLS = {
+	clientExtension:'/~/control_panel/manage?p_p_id=com_liferay_client_extension_web_internal_portlet_ClientExtensionAdminPortlet',
```
[the remark this answers is not in the document]
Comment (ilzamcmed) [reply modules/test/playwright/utils/portletUrls.ts:6]: > You probably forgot to remove it.

You know me so well... :laughing: 
Thanks @thiago-buarqque!

File: modules/test/playwright/pages/layout-admin-web/PagesAdminPage.ts [hunk 2]
```diff
@@ -66,4 +66,31 @@ export class PagesAdminPage {
...
+		);
+
+		await iframe.waitFor({
+			state: 'visible',
+		});
+
+		await this.page
+		.frameLocator('#selectGlobalJSCETs_iframe_').getByLabel(clientExtensionName).check();
+
+		await this.page.getByRole('button', {exact: true, name: 'Add' }).click();
```
Comment (ilzamcmed) [inline modules/test/playwright/pages/layout-admin-web/PagesAdminPage.ts:90]: @ethib137 Review this, please

File: modules/test/playwright/tests/client-extension-web/globalJS.spec.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+	clientExtensionsPageTest,
+	featureFlagsTest({
+		'LPD-10981': true,
+	}),
+	loginTest(),
+	pagesAdminPageTest,
+	globalJSClientExtensionsPageTest
+);
+
+test('Create a new JS with an attribute field', async ({
```
[the remark this answers is not in the document]
Comment (ilzamcmed) [reply modules/test/playwright/tests/client-extension-web/globalJS.spec.ts:25]: I'll add them, thanks @thiago-buarqque!

[the remark this answers is not in the document]
Comment (ilzamcmed) [reply modules/test/playwright/tests/client-extension-web/globalJS.spec.ts:25]: Actually, shouldn't this be tested in a unit test?

[the remark this answers is not in the document]
Comment (thiago-buarqque) [reply modules/test/playwright/tests/client-extension-web/globalJS.spec.ts:25]: You have a good point. The `src` test indeed can be a unit test. I'm not sure about checking about the other one (probably not).

Can you please convert the src test to a FE unit test and check if the other one can be converted as well?

**Comment by ethib137** [comment]
resent here: https://github.com/liferay-platform-experience/liferay-portal/pull/539

**Pull request closed without merging**